ensembler/system/basic_system.py

```diff
@@ -16,7 +16,7 @@
 from ensembler.util import dataStructure as data
 from ensembler.potentials._baseclasses import _potentialCls
 
-import ensembler.util.ensemblerTypes as ensemblerTypes
+from ensembler.util import  ensemblerTypes as ensemblerTypes
```

Comment:
This is still arkward, that the import system is very specific. @dfhahn  maybe we can find an solution to unified imports with the #14 cooke cutter structure?